src/adm/tiles.rs

```diff
@@ -65,21 +98,34 @@ pub async fn get_tiles(
     let adm_url = adm_url.as_str();
 
     trace!("get_tiles GET {}", adm_url);
-    let mut response: AdmTileResponse = reqwest_client
-        .get(adm_url)
-        .send()
-        .await
-        .map_err(|e| {
-            // ADM servers are down, or improperly configured
-            HandlerErrorKind::BadAdmResponse(format!("ADM Server Error: {:?}", e))
-        })?
-        .error_for_status()?
-        .json()
-        .await
-        .map_err(|e| {
-            // ADM servers are not returning correct information
-            HandlerErrorKind::BadAdmResponse(format!("ADM provided invalid response: {:?}", e))
-        })?;
+    let mut response: AdmTileResponse = if state.settings.test_mode {
+        let default = HeaderValue::from_str(DEFAULT).unwrap();
```

Comment:
That error pops when it can't find the test JSON file, so more than likely the `CONTILE_TEST_FILE_PATH` needed to be adjusted because you're not running in the project root. I added a better comments as well as normalized the file name to lower case to try and catch that.

Reply:
Ah, I misunderstood what was going on. Thanks for clarifying!